students/shibin_mathew/lesson6/mailroom.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+import sys
+from collections import defaultdict
+
+
+def print_donor_names(donors2):
+    list_test = []
+    for names in donors2:
+        print(names)
+        list_test.append(names)  # Debug - for unit testing
```

Comment:
You generally shouldn't have return values that are only for testing or debugging. But you could make the function more testable (and easier to debug) by returning a list to be printed where it is called, rather than printing the list as a side effect of the function